### PR TITLE
tests: fixed test errors in test_that and in script

### LINKED ISSUE #28: Boxplot test fixes

### PR DESCRIPTION
**Pull request by miyer26**

### CODE REVIEW

**Review by shyan0903 — APPROVED** [review]

File: tests/testthat/test-boxplot.R [hunk 1]
```diff
@@ -28,48 +28,44 @@ test_that("The check for input type for x is failing",{
...
+    expect_error(boxplot(df, origin, "miles_per_gallon"),
+                         "y, the column name of y, must be unquoted")
 })
 
 
 test_that("The check for input type for df is failing",{
-    expect_error(boxplot(c(1,2), miles_per_gallon, origin,
-                         "a data frame with at least 1 numerical and 1
-                          categorical column must be supplied"))
+    expect_error(boxplot(df$miles_per_gallon, miles_per_gallon, origin))
```
Comment (iamMoid) [inline tests/testthat/test-boxplot.R]: Did you remove the message by mistake?

**Review by iamMoid — COMMENTED** [review]
Just one comment

File: tests/testthat/test-boxplot.R (hunk 1 above)
Comment (miyer26) [reply tests/testthat/test-boxplot.R]: added it again, thank you!

**Review by ruben1dlg — APPROVED** [review]
Looks good! Once we merge we'll now if there are any changes pending 😀

**Review by iamMoid — APPROVED** [review]
Looks good.

**Pull request merged**